.github/workflows/acceptance-tests-runner.yml

```diff
@@ -331,12 +334,36 @@ jobs:
         with:
           terraform_version: ${{ inputs.terraform_version }}
           terraform_wrapper: false  
+      - name: Prepare new advanced_cluster
```

Comment:
The nightly tests now will invoke both these advancedcluster groups right?
If so, for a while we might have both tests creating cluster, could that make the Out of capacity issues worse or do you think we have that handled now?

Reply:
i think capacity issues are fixed now and it shouldn't be a problem

Reply:
understand are not a problem now, but aren't we doubling the amount of clusters? If so, I'd like to consider alternative solutions.

Reply:
We can exclude tests for advancedclulster TPF group from nightly runs and run only in PRs based on change detection in that package

Reply:
yeah, that's one option, only run them in PRs